### DISCUSSION TITLE
v0.65.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#3809] [Announcements]
This months release comes with a number of new features – full support for outgoing Webhooks so you can integrate Outline with other tools, Mermaid diagrams in the editor, and the ability to setup multiple authentication providers for your team. We also started a bug bounty program on [huntr.dev](https://huntr.dev/repos/outline/outline) which has resulted in an above-average number of security related patches in this release.

> **Note** It is recommended to visit **Settings -> Security** and add team domains as an additional measure to restrict access to your self-hosted instance. 

## Enhancements
* Publicly shared documents now render metadata on the server in https://github.com/outline/outline/pull/3646
* Personal Gmail accounts can now be used to sign into teams with an existing invite that matches by @thenanyu in https://github.com/outline/outline/pull/3652
* The `documents.update` API endpoint can now be used with collaborative editing enabled in https://github.com/outline/outline/pull/3647
* Mermaid.js support added to the editor in https://github.com/outline/outline/pull/3679
* Webhook support in https://github.com/outline/outline/pull/3691
* Added a new optional notification email when invite is accepted in https://github.com/outline/outline/pull/3718
* A new check for pending migrations during startup by @CuriousCorrelation in https://github.com/outline/outline/pull/3744
* Visiting an unpublished share link will now show a Login screen by @CuriousCorrelation in https://github.com/outline/outline/pull/3760
* Lists that failed to load will now show an error state and automatically retry in https://github.com/outline/outline/pull/3766
* Sign-in options for all configured authentication methods will now be displayed in https://github.com/outline/outline/pull/2986
* As always, new and improved translations from the community

### Platform
* Now possible to delete user via API in https://github.com/outline/outline/pull/3619
* Context menus now scrollable on iOS in https://github.com/outline/outline/pull/3626
* Lazily polyfill `ResizeObserver` for old iOS in https://github.com/outline/outline/pull/3629
* Google, Azure, OIDC SSO continued access are now verified on every app load in https://github.com/outline/outline/pull/3590
* Previously provisioned JWT's are now revoked on logout in https://github.com/outline/outline/pull/3639
* Don't show /share/ link in popover when team sharing disabled in https://github.com/outline/outline/pull/3714
* Localstorage cleared correctly after logout by @thenanyu in https://github.com/outline/outline/pull/3731
* Models are now all removed from local store upon access change in https://github.com/outline/outline/pull/3729
* Do not show suspended users to non admins in https://github.com/outline/outline/pull/3776
* Added additional model validations in https://github.com/outline/outline/pull/3725
* Improved websockets error handling in https://github.com/outline/outline/pull/3726
* Save button no longer shown when adding and removing a domain row by @paullessing in https://github.com/outline/outline/pull/3701
* Automatically cleanup api keys and webhooks for suspended users in https://github.com/outline/outline/pull/3756
* `svg+xml` image type ext not assigned properly by @CuriousCorrelation in https://github.com/outline/outline/pull/3774
* Fixed incorrect length validation with UTF-8 characters by @CuriousCorrelation in https://github.com/outline/outline/pull/3709

### Editor
* Cursor no longer disappears behind emoji by @CuriousCorrelation in https://github.com/outline/outline/pull/3786
* Moving an image to empty space no longer results in an endless upload in https://github.com/outline/outline/pull/3799
* Triple-click now selects the current line in codeblocks by @CuriousCorrelation in https://github.com/outline/outline/pull/3798
* The formatting toolbar now shows the correct options on inline code by @CuriousCorrelation in https://github.com/outline/outline/pull/3775
* Enter at beginning of collapsed heading now works correctly in https://github.com/outline/outline/pull/3754

## New Contributors
* @l3nnartt made their first contribution in https://github.com/outline/outline/pull/3596
* @CuriousCorrelation made their first contribution in https://github.com/outline/outline/pull/3709
* @paullessing made their first contribution in https://github.com/outline/outline/pull/3701
* @JamieSlome made their first contribution in https://github.com/outline/outline/pull/3711

**Full Changelog**: https://github.com/outline/outline/compare/v0.64.3...v0.65.0

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.65.0'>v0.65.0</a>.</em>

**Comment by bmm-alc** [comment]
and there is a follow-up release (0.65.1) for installation that does not use secure connection for database
